.github/workflows/ci.yml

```diff
@@ -11,6 +11,12 @@ on:
 jobs:
   ci:
     runs-on: ubuntu-latest
+    services:
```

Comment:
This is the container we run on CI that our tests connect to. 

For reference, it can be found [here](https://github.com/samui-build/samui-test-validator).